### PR TITLE
Add django-dynamic-form to the project

### PR DESCRIPTION
**Pull request by ZoltanOnody**
todo:
- [ ] regenerate the migrations before merging the PR
- [ ] write tests

### CODE REVIEW

File: questionnaires/views.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+from dynamic_forms.models import FormModel
+from dynamic_forms.views import DynamicFormView
+
+from konfera.models import Ticket
+
+from .models import FormForTicket
+
+
+class ShowFormForTicket(DynamicFormView):
```
Comment (ZoltanOnody) [inline questionnaires/views.py]: @ricco386 I am having some difficulties with this class based view.

when I visit `/form/1/ticket/my_uuid/` the `dispatch` is invoked as supposed but when I submit the form, `form_valid` is not invoked, however the `form_valid` in `DynamicFormView` is. So basically I cannot override the method and I have no idea why. Any suggestions?

Comment (ZoltanOnody) [inline questionnaires/views.py]: In the meantime I found out where is the problem.

File: questionnaires/requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+django-dynamic-forms-0.5.3
```
Comment (ricco386) [inline questionnaires/requirements.txt]: please fix the typo

File: requirements.txt [hunk 3]
```diff
@@ -4,3 +4,4 @@ django-sitetree==1.6.0
 mock==2.0.0
 pytz==2016.7
 Pillow==3.4.2
+django-dynamic-forms-0.5.3
```
Comment (ricco386) [inline requirements.txt]: please fix the typo

**Comment by ZoltanOnody** [comment]
As the app is going to be refactored and simplified (#320), I close this PR.

**Pull request closed without merging**